package.json

```diff
@@ -44,6 +44,8 @@
 	"dependencies": {
 		"ajv": "6.7.0",
 		"async": "2.6.1",
+		"axon": "^2.0.3",
```

Comment:
Are we using the `^` notation?

Reply:
Good catch. It was copied from `lisk-modular` but shouldn't be used. I will remove it ;)